frontend/src/assets/content/arp/map-alt-text/red-feather-lakes.md

```diff
@@ -9,36 +9,31 @@ Map Key
+The Pay Station is on County Road 69 which is 3.25 Miles north of the junction with Vixen Way. 
 
-Smokey Bear's Lane is the first cutting area road that is one way towards the west and is 1.4 miles long. The junction for Donner way is about half way down Smokey Bear lane, is a one-way road to the south and west and ties into Comet lane.
 
-The Smokey Bear* stop is at the end of the road at Dasher's Way intersection. It has First Aid and toilets available. 
+Entrance Station (an asterisk is noted) is located 3.35 miles southeast of the junction of Vixen Way and County Road 69.
+Entrance Station is located 1.60 miles east of the Christmas tree area entrance on County Road 69. It is at the intersection with County Road 68C at Goodell Corner. There is a chain-up area and toilets. County Road 69 continues south 3.20 Miles to Rustic and Hwy 14.
 
 
-Comet's Lane is a one-way cutting area road that goes south west and loops into Dasher's Way after 1.3 miles.
+The cutting area is .50 miles from the Entrance marked 171.A. The entrance is one-way only. The gate closes at 2:30.
 
-The intersection of Rudolph's Way is .20 miles northwest from Smokey Bear. Rudolph's Way is a one-way cutting area road going northeast for .85 Miles until it merges into Vixen's Way, which is .70 miles to the Exit.
+Smokey Bear Lane is the first cutting area road that is one way and west bound. It is 1.4 miles long. It ends at a junction with Dasher Way. Dasher Way is one-way and heads east and south between Blitzen Lane and Smokey Bear Lane before intersecting with Comet Lane. The junction for Donner way is about half way down Smokey Bear lane, is a one-way road to the south and west and ties into Comet Lane which is also a one-way headed west and north. Coment Lane loops into Dasher Way after 1.3 miles. Continuing onto Dasher Way from Comet Lane, the junction for Blitzen Lane is to the north. Blitzen Lane is a one-way road to the north, and ties into Vixen Way after 1.5 miles. from the junction of Blitzen Lane and Vixen Way, it is 0.70 miles on Vixen Way to the northeast to County Road 69.
 
-Dasher's Way is a one-way road that goes east for .75 miles from the end of Comet's Lane.
-Blitzen's Lane is a one-way road that forks northeast from the start of Dasher's Way and heads toward the exit with 1.5 miles until Vixen's Way.
+The Smokey Bear (an asterisk is noted) stop is at the intersection of Dasher Way and Smokey Bear Lane. It has toilets available. 
 
-Please return card at the pay station.
+Rudolph Way is a one-way road that will take you from Dasher Way north to Vixen Way. It is 0.85 miles. The intersection of Rudolph Way is 0.20 miles northwest from the Smokey Bear stop.
 
 
-Note: Smokey Bear, Pay Station and Entrance Station are staffed only on weekends. This map is not to scale.
+Asterisk Note at the bottom of the map reads: Smokey Bear, Entrance Station, and Pay Station are available only on weekends. This map is not to scale.
```

Comment:
Should this be a literal asterisk? `\*` ?

Reply:
No, it is a description of what is on the map.  It was intentional to write out "asterisk note" here so that it is associated with the other "asterisk" comments in the alt text.